NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local-packages" value="./libs/net/packages" />
```

Comment:
This should allow all our local environments build the projects with the local packages.  I am not certain this will work in other environments (Github, Openshift) yet.

Reply:
why do we change to local nuget? project reference doesn't work in .net 8.0?